src/index.js

```diff
@@ -0,0 +1,88 @@
+import { helpers } from '@eik/common';
+import { join } from 'path';
+import fetch from 'node-fetch';
+
+const isUrl = (value = '') => value.startsWith('http');
+
+const fetchImportMaps = async (urls = []) => {
+    try {
+        const maps = urls.map((map) => fetch(map).then((result) => {
+            if (result.status === 404) {
+                throw new Error('Import map could not be found on server');
+            } else if (result.status >= 400 && result.status < 500) {
+                throw new Error('Server rejected client request');
+            } else if (result.status >= 500) {
+                throw new Error('Server error');
+            }
+            return result.json();
+        }));
+        return await Promise.all(maps);
+    } catch (err) {
+        throw new Error(
+            `Unable to load import map file from server: ${err.message}`,
+        );
+    }
+};
+
+export default class EikNodeClient {
+    constructor({
+        development = false,
+        base = '',
+        path = process.cwd(),
```

Comment:
How will this work in a monorepo structure? If I run my code from root, will the client know of the folders inside package A.

```
/
package.json -> script that builds package A
packages/
    layouts/
        packageA/
            package.json
    podlets/
        packageB
```

Reply:
So, this is for reading the EIk config during run time (not build time). If `packageA` is an app (http server or so), how is it started in local development and how is it deployed to a production environment and started there?

Reply:
There are times when I make scripts in root to run the development server in ex. package A.

For example from root:
`yarn workspace packageA dev`

Reply:
nevermind, did a small test locally and it works as I wanted.

Reply:
Cool